docker/nuxeo-explorer-docker/pom.xml

```diff
@@ -64,7 +64,6 @@
               <pullNewerImage>false</pullNewerImage>
               <buildArgs>
                 <BASE_IMAGE>docker-private.packages.nuxeo.com/nuxeo/nuxeo:${nuxeo.image.version}</BASE_IMAGE>
-                <NUXEO_IMAGE_VERSION>${nuxeo.image.version}</NUXEO_IMAGE_VERSION>
```

Comment:
Isn't it needed by the maven build as the scaffold for this image use it?

Reply:
I don't think so, it's not used in the Dockerfile, only by Skaffold for which it is handled by `envsubst` in the pipeline.